Include networkmanager header after the musl ones, to avoid conflicts.

diff --git a/panels/network/wireless-security/utils.c b/panels/network/wireless-security/utils.c
index b74424d..2bb543f 100644
--- a/panels/network/wireless-security/utils.c
+++ b/panels/network/wireless-security/utils.c
@@ -20,11 +20,10 @@
  * Copyright 2007 - 2015 Red Hat, Inc.
  */
 
-#include "nm-default.h"
-
 #include <string.h>
 #include <netinet/ether.h>
 
+#include "nm-default.h"
 #include "utils.h"
 
 /**
